LectureOne/09-fishTank.js

```diff
@@ -0,0 +1,14 @@
+function fishTank(input){
+    let aquariumLengthcInCentimeters = Number(input[0]);
```

Comment:
You could use const instead of let almost everywhere. It's a good practice that a function returns a value, you still can console log it before you return

Ex:
function basketballEquipment(input){

    const annualTrainingFee = Number(input[0]);

    const basketballsneakers = annualTrainingFee - annualTrainingFee * 0.40;
    const basketballTshirt = basketballsneakers - basketballsneakers * 0.20;
    const basketballBall = basketballTshirt * 0.25;
    const basketballAccessories = basketballBall * 0.20;

    const totalsum = annualTrainingFee + basketballsneakers + basketballTshirt + basketballBall + basketballAccessories;

    console.log(totalsum);
    return totalsum;
}